num_sensors: 2
sensor_0:
  naming:
    name:  Xtion
    topic: depth_pcl_filtered
    frame: camera_rgb_optical_frame
    frame_explore: camera_link
  properties:
    min_range:     0.5
    max_range:     4.0
    num_max_skip:  9
    sigma:         0.1
    prob_hit:      0.9
    prob_rand:     0.1
    prob_max_free: 0.0001
    prob_max_occ:  0.9999
    prob_det_sat:  0.0
    consider_max:  true
    has_colors:    true
  print_timers:    true
sensor_1:
  naming:
    name:  Hokuyo
    topic: depth_lsr
    frame: laser
    frame_explore: laser
  properties:
    min_range:     0.5
    max_range:     4.0
    num_max_skip:  0
    sigma:         0.1
    prob_hit:      0.9
    prob_rand:     0.1
    prob_max_free: 0.0001
    prob_max_occ:  0.9999
    prob_det_sat:  0.0
    consider_max:  true
    has_colors:    false    
  print_timers:    true
sensor_2:
  naming:
    name:  FicticiousXtion
    frame: camera_rgb_optical_frame
    frame_explore: camera_link
  properties:
    min_range:     0.5
    max_range:     3.5
    sigma:         0.05
    prob_hit:      0.9
    prob_rand:     0.1
    prob_max_free: 0.01
    prob_max_occ:  0.99
    prob_det_sat:  0.0
    consider_max:  false
    has_colors:    false
